### PR TITLE
chore(setupproject):set up empty express bolier plate project with do…

### PR DESCRIPTION
**Pull request by bourgiworks**
Link Pivotal Tracker with Github

### CODE REVIEW

**Comment by munyanezaarmel** [comment]
**What does this PR do?**
it set up all folders required for backend 

**how should this be manually tested**
after cloning a repo cd into it and RUN npm run start

**any background context you want to provide**
all JavaScript are written in ES6 and it use babel to transpire down to ES5

**Comment by munyanezaarmel** [comment]
**What does this PR do?**
it set up all folders required for backend 

**how should this be manually tested**
after cloning a repo cd into it and RUN npm run start

**any background context you want to provide**
all JavaScript are written in ES6 and it use babel to transpire down to ES5

**what are the relevant pivotal tracker stories**
#182201507

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import express from 'express'
+import passport from 'passport'
+//import * as babel from '@babel/core';
+let app=express()
+require('dotenv').config()
```
Comment (Ntare22) [inline src/index.js:5]: Hi @munyanezaarmel,

This line of code is not an ES6 line of code, please make adjustment

File: package.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  "description": "",
+  "main": "index.js",
+  "type":"module",
+  "scripts": {
+    "start": " node ./src/index.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/munyanezaarmel/elite-bn-be.git"
+  },
```
Comment (Ntare22) [inline package.json:13]: I don't see babel setup?

**Review by Ntare22 — CHANGES_REQUESTED** [review]
Please setup babel & correct ES5

**Pull request closed without merging**